src/tests/Feature/ComplaintTest.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Models\Complaint;
+use App\Models\User;
+use Tests\BaseFeatureTest;
+
+class ComplaintTest extends BaseFeatureTest
+{
+    protected string $uri;
+
+    /**
+     * @return void
+     */
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->uri = self::BASE_URI . '/complaints';
+    }
+
+    /** @test */
+    public function it_should_not_get_complaints_when_does_not_have_permission()
+    {
+        $complain = Complaint::factory()->create();
+
+        $user = User::factory()->create();
+
+        $response = $this->actingAs($user)->json('GET', $this->uri . '/' . $complain->id);
+
+        $response->assertForbidden();
+    }
+
+    /** @test */
+    public function it_should_get_complaints_when_has_permission()
+    {
+        $complain = Complaint::factory()->create();
+
+        $user = User::factory()->create()->givePermissionTo('complaints.view');
+
+        $response = $this->actingAs($user)->json('GET', $this->uri . '/' . $complain->id);
+
+        $response->assertOk();
+        $response->assertJsonFragment(['complaint' => $complain->load('createdUser', 'reviewedUser', 'relatedUser')->toArray()]);
+    }
+
+    /** @test */
+    public function it_should_get_complaints_when_created_user_requested()
+    {
+        $complain = Complaint::factory()->create();
+
+        $response = $this->actingAs($complain->createdUser)->json('GET', $this->uri . '/' . $complain->id);
+
+        $response->assertOk();
+        $response->assertJsonFragment(['complaint' => $complain->load('createdUser', 'reviewedUser', 'relatedUser')->toArray()]);
+    }
+
+    /** @test */
+    public function it_should_not_update_complain_when_does_not_have_permission()
+    {
+        $complain = Complaint::factory()->create();
+
+        $user = User::factory()->create();
+
+        $response = $this->actingAs($user)->json('PUT', $this->uri . '/' . $complain->id, [
+            'description' => 'new description',
+        ]);
```

Comment:
Hocam şu yeni satıra kırmalar için şöyle bi yol izlesek ne dersin?
en dış katmandan içe doğru  120 karaktere sığana kadar kırsak
bu şekilde hangi method aşağıda hangisi yukarda hangisi yeni parametre vs anlamak baya uğraştırıyor.

Görselde örnek olarak, üstteki yerine alttaki gibi uygulasak okunurluk artmaz mı?

![image](https://user-images.githubusercontent.com/6912824/223724531-2abcbb44-fe51-4fb2-93e0-3aa5c9ddabf0.png)